experimenter/tests/integration/nimbus/utils/filter_expression.js

```diff
@@ -1,55 +1,64 @@
-async function remoteSettings(arguments) {
-
+async function remoteSettings(targetingString, recipe, callback) {
     /*
-        Arguments contains 2 items.
-        arguments[0] - the JEXL targeting string
-        arguments[1] - the experiment recipe
+    Arguments contains 2 items.
+    arguments[0] - the JEXL targeting string
+    arguments[1] - the experiment recipe
     */
 
     // See https://bugzilla.mozilla.org/show_bug.cgi?id=1868838 -
     // ASRouterTargeting was moved from browser/components/newtab into
     // browser/components/asrouter and its import path changed.
-    const { TelemetryEnvironment } = ChromeUtils.importESModule("resource://gre/modules/TelemetryEnvironment.sys.mjs");
-    await TelemetryEnvironment.onInitialized();
-
-    let ASRouterTargeting;
 
     try {
-        ASRouterTargeting = ChromeUtils.importESModule("resource:///modules/asrouter/ASRouterTargeting.sys.mjs");
-    } catch (ex) {
-        if (ex.result === Cr.NS_ERROR_FILE_NOT_FOUND) {
-            ASRouterTargeting = ChromeUtils.importESModule("resource://activity-stream/lib/ASRouterTargeting.sys.mjs");
-        } else {
-            throw ex;
+        const { TelemetryEnvironment } = ChromeUtils.importESModule("resource://gre/modules/TelemetryEnvironment.sys.mjs");
+        await TelemetryEnvironment.onInitialized();
+
+        let ASRouterTargeting;
+        try {
+            ASRouterTargeting = ChromeUtils.importESModule("resource:///modules/asrouter/ASRouterTargeting.sys.mjs");
+        } catch (ex) {
+            if (ex.result === Cr.NS_ERROR_FILE_NOT_FOUND) {
+                ASRouterTargeting = ChromeUtils.importESModule("resource://activity-stream/lib/ASRouterTargeting.sys.mjs");
+            } else {
+                throw ex;
+            }
         }
```

Comment:
We're on 139+ so we can drop the try/catch and import from modules/asrouter directly.

Doesn't have to be in this change tho.